ORMSolution/ORM/ORM/PrimaryKey/ORMPrimaryKey.cs

```diff
@@ -32,7 +32,7 @@ internal ORMPrimaryKey(IDataReader reader, int[] primaryKeyIndexes)
             var hashCode = new HashCode();
             for (int i = 0; i < primaryKeyIndexes.Length; i++)
             {
-                Add(reader.GetName(i), reader.GetValue(i));
+                Add(string.Empty, reader.GetName(i), reader.GetValue(i));
```

Comment:
Waarom voeg je het hier toe als een string.empty by default?

Reply:
Hm zit net even terug te kijken, maar de constructie is sowieso wat vreemd nu (met oog op het vullen in de constructor van de ORMEntity). Ik ga hier nog even naar kijken.

Reply:
Fix before 0.2